TeensyDmx.h

```diff
@@ -146,6 +146,8 @@ class TeensyDmx
     void startReceive();
     void stopReceive();
 
+    inline void setDirection(bool transmit);
```

Comment:
inline is superfluous here.  Should only be written at method implementation.

Reply:
Fixed, and a bit more tidying added too.